engine/src/mac-dialog.mm

```diff
@@ -646,12 +646,6 @@ void MCPlatformBeginFileDialog(MCPlatformFileDialogKind p_kind, MCPlatformWindow
 	else 
 		[t_panel setCanCreateDirectories: YES];
 	
-	if ([t_panel respondsToSelector:@selector(isAccessoryViewDisclosed)])
```

Comment:
Don't we need this block for backwards compatibility? Could we instead do something like:
```
if ([t_panel isMemberOfClass:[NSOpenPanel class]] && [t_panel respondsToSelector:@selector(isAccessoryViewDisclosed)])
```